### PR TITLE
Adding topic, resource type and persona pills on stand alone pages

### PR DESCRIPTION
**Pull request by DerekSgov**
## Summary
- Adding header on stand alone pages like the one on topic pages
- Adding a pill with icon for resource type
- Adding pill(s) with topic icon for topics which the stand alone resource is in
- Changing path of about page to not interfere with resource called About

### CODE REVIEW

File: app-web/src/templates/StandAlone_GitHubRaw.js [hunk 1]
```diff
@@ -20,30 +20,71 @@ import React from 'react';
...
 
 export const StandAloneGitHubRawResource = ({ data: { githubRaw } }) => {
-  const isExternal = false;
   const renderAst = new rehypeReact({
     createElement: React.createElement,
   }).Compiler;
 
+  let topics = githubRaw.fields.topics;
+  let topicPills = [];
+  if (topics) {
```
Comment (patricksimonian) [inline app-web/src/templates/StandAlone_GitHubRaw.js]: so cool!! :D

File: app-web/src/templates/StandAlone_GitHubRaw.js [hunk 2]
```diff
@@ -20,30 +20,71 @@ import React from 'react';
...
+    topicPills = topics.map(topic => {
+      return (
+        <a href={`/${topic.fields.slug}/${githubRaw.fields.slug}`} key={topic.id}>
```
Comment (patricksimonian) [inline app-web/src/templates/StandAlone_GitHubRaw.js]: use the `Link` Component for non web uris

File: app-web/src/templates/StandAlone_GitHubRaw.js [hunk 3]
```diff
@@ -20,30 +20,71 @@ import React from 'react';
...
+          <IconPill topic={true} label={topic.name} variant="filled" deletable={false} />
+        </a>
+      );
+    });
+  }
+
   return (
     <Layout>
+      <Header>
+        <a href={`/${slugify(githubRaw.fields.resourceType).toLowerCase()}`} key={githubRaw.id}>
```
Comment (patricksimonian) [inline app-web/src/templates/StandAlone_GitHubRaw.js]: use `Link` Component

File: app-web/src/templates/StandAlone_GitHubRaw.js (hunk 1 above)
Comment (DerekSgov) [reply app-web/src/templates/StandAlone_GitHubRaw.js]: Thanks!!

File: app-web/src/templates/StandAlone_GitHubRaw.js (hunk 2 above)
Comment (DerekSgov) [reply app-web/src/templates/StandAlone_GitHubRaw.js]: good call, consider it done

Comment (patricksimonian) [inline app-web/src/templates/StandAlone_GitHubRaw.js]: still not `Link` component

File: app-web/src/templates/StandAlone_GitHubRaw.js [hunk 4]
```diff
@@ -20,30 +20,130 @@ import React from 'react';
...
+
 export const StandAloneGitHubRawResource = ({ data: { githubRaw } }) => {
-  const isExternal = false;
   const renderAst = new rehypeReact({
     createElement: React.createElement,
   }).Compiler;
 
+  //create pill to show resource type and links out to the corresponding resource type page
+  let resourceTypePill = (
+    <PillLink href={`/${slugify(githubRaw.fields.resourceType).toLowerCase()}`} key={githubRaw.id}>
```
Comment (patricksimonian) [inline app-web/src/templates/StandAlone_GitHubRaw.js]: so close! `<Link to={...} />` not `<Link href={..} />`

**Pull request merged**